setup.py

```diff
@@ -53,11 +53,11 @@
     'scipy>=0.14.0',
     'seaborn>=0.7.1',
     'pandas-datareader>=0.2',
-    'empyrical>=0.2.2'
+    'empyrical>=0.3.0'
 ]
 
```

Comment:
Do we also want to pin versions instead of using `>=`? The current pandas issues were due to this, but there are advantages/problems with both `>=` and `==`.